web/src/classic/components/molecules/Visualizer/Block/HTML/index.stories.tsx

```diff
@@ -3,6 +3,7 @@ import { Meta, Story } from "@storybook/react";
 import HTML, { Props } from ".";
 
 export default {
+  title: "molecules/Visualizer/Block/HTML",
```

Comment:
why add title here?

Reply:
To keep it organized correctly. Without this, it becomes classic/xx/xx/xx and all other classic stories have their title explicitly defined.
In beta/ we will let storybook infer titles

Reply:
Yeah, Makes sense.